flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/MySqlToPaimonE2eITCase.java

```diff
@@ -0,0 +1,371 @@
+                                0.875f,
+                                "red",
+                                "{\"k1\": \"v1\", \"k2\": \"v2\"}"),
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                106,
+                                "hammer",
+                                "18oz carpenter hammer",
+                                1.0f,
+                                null,
+                                null),
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                107,
+                                "rocks",
+                                "box of assorted rocks",
+                                5.1f,
+                                null,
+                                null),
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                108,
+                                "jacket",
+                                "water resistent black wind breaker",
+                                0.1f,
+                                null,
+                                null),
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                109,
+                                "spare tire",
+                                "24 inch spare tire",
+                                22.2f,
+                                null,
+                                null),
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                110,
+                                "jacket",
+                                "water resistent white wind breaker",
+                                0.2f,
+                                null,
+                                null),
+                        Row.ofKind(
+                                RowKind.INSERT,
+                                111,
+                                "scooter",
+                                "Big 2-wheel scooter ",
+                                5.18f,
+                                null,
+                                null),
+                        Row.ofKind(RowKind.INSERT, 112, "finally", null, 2.14f, null, null)));
+    }
+
+    private void validateSinkResult(String query, List<Row> expected) {
+        long startWaitingTimestamp = System.currentTimeMillis();
+        boolean validateSucceed = false;
+        while (!validateSucceed) {
+            try {
+                List<Row> results = new ArrayList<>();
+                tEnv.executeSql(query).collect().forEachRemaining(results::add);
+                assertEqualsInAnyOrder(expected, results);
+                validateSucceed = true;
+            } catch (Throwable e) {
+                if (System.currentTimeMillis() - startWaitingTimestamp
+                        > TESTCASE_TIMEOUT_SECONDS * 1000_1000L) {
```

Comment:
```suggestion
                        > TESTCASE_TIMEOUT_SECONDS * 1000L) {
```

`60 * 1000_1000L`ms is nearly 7 days